src/style.css

```
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.star_stars:not(:last-child) {
```

```suggestion
.stars__star:not(:last-child) {
```
also check other styles.